src/examples/tags/TagsWeight.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Span } from '@zendeskgarden/react-typography';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row>
+    <Col textAlign="center">
+      <Tag isRegular>
+        <span>
+          <Span isBold>Category</Span> Algae
+        </span>
+        <Tag.Close />
+      </Tag>
+    </Col>
+    <Col textAlign="center">
+      <Tag isRegular hue="royalblue">
+        <span>
+          <b>Category</b> Moss
```

Comment:
Please stick with Garden here, using `<Span isBold>Category</Span> Moss`. We get expected semibold styling on `<b>` only when `css-bedrock` is in the mix, otherwise `<b>` maps to bold, which is off-pattern for Garden.